app/controllers/api/v0/sources_controller.rb

```diff
@@ -6,6 +6,13 @@ class SourcesController < ApplicationController
       include Api::V0::Mixins::ShowMixin
       include Api::V0::Mixins::UpdateMixin
 
+      def set_the_current_tenant
+        # Auto-create tenant if creating a new source and tenant does not exist
+        Tenant.find_or_create_by(:external_tenant => user_identity) if action_name == "create"
```

Comment:
why is user_identity stored in the external_tenant?

Reply:
It's the account number. I'll rename that method